### DISCUSSION TITLE
Welcome to Albert Kermin Industries (AKI) Discussions!

### DISCUSSION THREAD

**Discussion by zer0Kerbal** [zer0Kerbal/StockalikeRandomRocketParts#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by AtomikkuSan** [comment]
Another new parts pack that needs tweaking?
*Laughs in AtomicTech*
